File: src/prime_normal.cc
#include <iostream>

int findPrime(int min, int max) {
    int count, num, c = 0;
    for(int i=min; i<=max; i++)
        {
        num = i;
        count = 0;  
        for(int x = 1; x <= num; x++) {
            if(num%x == 0) {
                count++;
            }
            }
        if(count == 2) {
            c++;
        }  
    }

    return c;
}

int main() {
    int max = 100;
    int count = findPrime(2, max);

    std::cout << count << "\n";
    return 0;
}

File: src/prime_normal.rs
pub fn find_prime(min: i32, max: i32) -> i32 {
    let mut count: i32;
    let mut num: i32;
    let mut c: i32 = 0;

    for i in min..=max {
        num = i;
        count = 0;
        for x in 1..=num {
            if num%x == 0 {
                count+=1;
            }
        }
        if count == 2 {
            c+=1;
        }
    }

    return c;
}